opentelemetry-userevents-exporter/examples/basic.rs

```diff
@@ -0,0 +1,55 @@
+#![allow(unused_imports, unused_mut, unused_variables, dead_code)]
+
+//! run with `$ cargo run --example basic --all-features
+
+use opentelemetry_api::{
+    logs::LoggerProvider as _,
+    logs::LogRecordBuilder,
+    logs::Logger,
+    logs::Severity,
+    trace::{Span, Tracer, TracerProvider as _, mark_span_as_active},
+    Context, KeyValue, logs::AnyValue
+};
+
+use opentelemetry_sdk::{
+    logs::LoggerProvider,
+    runtime,
+    trace::TracerProvider,
+};
+
+use opentelemetry_userevents_exporter_log::{ExporterConfig, RealTimeLogProcessor, ProviderGroup};
+use std::time::SystemTime;
+use std::thread;
+
+fn init_logger() -> LoggerProvider {
+    let exporter_config = ExporterConfig{keyword : 1};
+    let prov_group = Some("prov_group");
+    let realtime_processor = RealTimeLogProcessor::new("test12345", None , exporter_config);
```

Comment:
its not obvious from here that the processor creates a exporter inside it. Perhaps its best to make it explicit by creating the exporter right here, and feed it to the processor.